src/data/cassandra/impl/Result.h

```diff
@@ -86,23 +86,23 @@ extractColumn(CassRow const* row, std::size_t idx)
     }
     else if constexpr (std::is_convertible_v<DecayedType, std::string>)
     {
-        char const* value;
-        std::size_t len;
+        char const* value = nullptr;
+        std::size_t len = 0;
         auto const rc = cass_value_get_string(cass_row_get_column(row, idx), &value, &len);
         throwErrorIfNeeded(rc, "Extract string");
         output = std::string{value, len};
     }
     else if constexpr (std::is_same_v<DecayedType, bool>)
     {
-        cass_bool_t flag;
+        cass_bool_t flag = cass_bool_t::cass_false;
         auto const rc = cass_value_get_bool(cass_row_get_column(row, idx), &flag);
         throwErrorIfNeeded(rc, "Extract bool");
-        output = flag ? true : false;
+        output = flag != 0;
```

Comment:
Since flag is cass_bool_t, maybe compare with cass_bool_t::cass_false is more readable.